### PR TITLE
DM-12957: LDM-503-2 test report

### PR DESCRIPTION
**Pull request by jdswinbank**

### CODE REVIEW

**Comment by jdswinbank** [comment]
Also incorporating DM-12821.

File: DMTR-51.tex [hunk 1]
```diff
@@ -1,240 +1,578 @@
...
-DRP-00-15 & pass/fail & comment \\\hline
-DRP-00-20 & pass/fail & comment \\\hline
-DRP-00-25 & pass/fail & comment \\\hline
-DRP-00-30 & pass/fail & comment \\\hline
-DRP-00-35 & pass/fail & comment \\\hline
+DRP-00-00 & Pass & Refer to \S\ref{sect:detail-drp-00-00}. \\\hline
+DRP-00-05 & Pass & Refer to \S\ref{sect:detail-drp-00-05}. \\\hline
+DRP-00-10 & Pass & Refer to \S\ref{sect:detail-drp-00-10}. \\\hline
+DRP-00-15 & Pass & Refer to \S\ref{sect:detail-drp-00-15}. \\\hline
+DRP-00-25 & Partial Pass & An issue was identified when comparing photometric measurements to the reference catalog. Refer to \S\ref{sect:detail-drp-00-25}.\\\hline
```
Comment (TallJimbo) [inline DMTR-51.tex]: Should this be "An issue was identified in aperture corrections when comparing different photometry algorithms"?

That would be more consistent with the text in the next subsection, and I think that's the only issue we found that's large enough to be a priority now (we did also find problems when comparing our photometry to that of the reference catalog, but while significant, the bias there was smaller than the statistical errors).

File: DMTR-51.tex [hunk 2]
```diff
@@ -1,240 +1,578 @@
...
+It was noted that:
+
 \begin{itemize}
-\item A description of the deviation (e.g. substitution of any resource, procedural step not followed, scchedule deviation, etc)
-\item The rationale of the deviation
-\item An assessment of the deviation impact on the validity of the test case
+
+  \item{We note that the background model is stored separately from the image, but is readily accessible through the same mechanism;}
+  \item{
+    Some cosmic rays were not being properly masked.
```
Comment (TallJimbo) [inline DMTR-51.tex]: More precisely, I'd say "Pixels marked as contaminated by cosmic rays that had been successfully interpolated were not being included in the coadd, unnecessarily making much of the coadd unusable for precision science."

**Review by TallJimbo — APPROVED** [review]

File: DMTR-51.tex [hunk 3]
```diff
@@ -1,240 +1,578 @@
...
+    \begin{itemize}
+      \item{This offset is regarded as falling within normal tolerances, but worthy of further investigation.}
+      \item{Ticket \jira{DM-13057} has been filed.}
+    \end{itemize}
+  }
+
 \end{itemize}
 
-\paragraph{[DRP-00-05]}
+The test team agreed that all measurements feel within acceptable tolerances, and therefore regard the test case as having been passed. \jira{DM-13056} and \jira{DM-13057} will be scheduled for further investigation as part of the regular development cycle.
```
Comment (laurenam) [inline DMTR-51.tex]: feel --> fell (or fall?)

"...as having been passed."
- remove "been"?

File: DMTR-51.tex [hunk 4]
```diff
@@ -1,240 +1,578 @@
...
+  \item{Forced and unforced measurements are consistent.}
+  \item{The slope of the stellar locus appears appropriate.}
+
 \end{itemize}
 
-\subsubsection{Deviations from test cases/procedures}
-\paragraph{[TEST-CASE-ID]}
-This paragraph shall identify by their unique identification, the test cases in which one or more deviations occured and shall provide:
+However, aperture corrections were not found to be consistent between photometry algorithms.
+This inconsistency was outside acceptable tolerances, and has been ticketed for further investigation as \jira{DM-13058}.
```
Comment (laurenam) [inline DMTR-51.tex]: - now identified as a pathology in the forced catalogues only

File: DMTR-51.tex [hunk 5]
```diff
@@ -1,240 +1,578 @@
...
+
+This test case is regarded as ``partially passed'': its overall goal has been accomplished, but understanding the origin of the inconsistent aperture corrections will require further work which will be undertaken during the upcoming developing cycle.
+
+\subsection{DRP-00-30}
+\label{sect:detail-drp-00-30}
+
+The input dataset used for this test case were the same as for DRP-00-15 (see \S\ref{sect:detail-drp-00-15}).
```
Comment (laurenam) [inline DMTR-51.tex]: "The input dataset...*was* the same" (i.e. dataset is singular)

File: DMTR-51.tex [hunk 6]
```diff
@@ -1,240 +1,578 @@
...
+The test was carried out as described.
+All required artefacts were accessible.
+Data was inspected by the Data Release Production Science Lead, who certified that artifacts were correctly matched and that backgrounds were not over-subtracted.
+
+This test case is therefore successfully passed.
+
+\subsection{DRP-00-35}
+\label{sect:detail-drp-00-35}
+
+The input dataset used for this test case were the same as for DRP-00-15 (see \S\ref{sect:detail-drp-00-15}).
```
Comment (laurenam) [inline DMTR-51.tex]: ditto (were --> was)

File: DMTR-51.tex [hunk 7]
```diff
@@ -1,240 +1,578 @@
...
+
+The test was carried out as described.
+All required artefacts were accessible.
+Data was inspected by the Data Release Production Science Lead, who certified that artifacts were correctly matched, satellite trails and ghosts were properly rejected, and that backgrounds were not over-subtracted.
```
Comment (laurenam) [inline DMTR-51.tex]: Inconsistent art*e*facts vs. art*i*facts (throughout)

File: DMTR-51.tex [hunk 8]
```diff
@@ -1,240 +1,578 @@
...
+
+It was noted that:
+
 \begin{itemize}
-\item A description of the deviation (e.g. substitution of any resource, procedural step not followed, scchedule deviation, etc)
-\item The rationale of the deviation
-\item An assessment of the deviation impact on the validity of the test case
+
+  \item{We note that the background model is stored separately from the image, but is readily accessible through the same mechanism;}
```
Comment (laurenam) [inline DMTR-51.tex]: It's not obvious to me what "mechanism" is being referred to here.

File: DMTR-51.tex (hunk 1 above)
Comment (jdswinbank) [reply DMTR-51.tex]: Well spotted; thanks. My first draft flagged up the wrong problem as a priority — I noticed and fixed it, but obviously didn't correct the wording in this table.

File: DMTR-51.tex (hunk 8 above)
Comment (jdswinbank) [reply DMTR-51.tex]: The mechanism used to access the image. :-) But yeah, point taken.

**Pull request merged**